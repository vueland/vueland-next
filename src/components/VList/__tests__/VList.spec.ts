import { mount, VueWrapper } from "@vue/test-utils";
import { VList } from "../index";

describe("VList", () => {
  let mountFunction: (options?: any) => VueWrapper<any>;

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VList, { ...options });
  });

  it("should mount component and match snapshot", () => {
    const cmp = mountFunction();

    expect(cmp.html()).toMatchSnapshot();
  });
});
