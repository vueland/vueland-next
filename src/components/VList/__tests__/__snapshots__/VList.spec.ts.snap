// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VList should mount component and match snapshot 1`] = `
<div class="v-list"
     style="background-color: rgb(255, 255, 255); border-color: #ffffff;"
>
  <div class="v-list-item">
    first
  </div>
  <div class="v-list-item">
    second
  </div>
</div>
`;
